.github/workflows/buildkit.yml

```diff
@@ -13,7 +13,7 @@ on:
   pull_request:
 
 env:
-  GO_VERSION: "1.21.10"
+  GO_VERSION: "1.21.12"
```

Comment:
```suggestion
  GO_VERSION: "1.21.12"
  ALPINE_VERSION: "3.19"
```

Reply:
+1, thanks @corhere for the clarification!